gitlint/configs/pylintrc

```diff
@@ -38,4 +38,4 @@ docstring-min-length=10
 
 notes=FIXME,TODO,XXX
 
-good-names=setUpModule,tearDownModule,app,pool,i,j,k,ex,e,ip,ob,_,__,id,df
```

Comment:
Please remove this unintended diff

Reply:
holy cow its impossible to do in github edit

Reply:
holy cow its impossible to do in github edit